TERMS.md

```diff
@@ -544,6 +544,12 @@ A set of conditions, internals, and actions that create notifications.
 
 Avoid using. Use *separate* or *isolate* instead.
 
+**setting**
+
+Refers to the key-value pairs that create mappings in the various YAML configuration files used throughout OpenSearch. Sometimes alternatively called parameters, the programming language manipulating the key-value pair usually dictates the name of this mapping in a YAML file. For OpenSearch documentation (java) they are properly a `Setting` object.
```

Comment:
a couple of optional suggestions...

"A key-value pair that creates a mapping in the . . . "
"For example, with Java, you can create a`Settings` object that takes a key-value pair, such as `Settings.index.number_of_shards` set to a value of 4."

Reply:
+1